src/algebra/lie_algebra.lean

```diff
@@ -380,6 +380,23 @@ abbreviation mk : M → N.quotient := submodule.quotient.mk
 lemma is_quotient_mk (m : M) :
   quotient.mk' m = (mk m : N.quotient) := rfl
 
+/-- Given a Lie module $M$ over a Lie algebra $L$, together with a Lie submodule $N ⊆ M$, there
+is a natural linear map from $L$ to the endomorphisms of $M$ leaving $N$ invariant. -/
+def lie_submodule_invariant : L →ₗ[R] submodule.comap_linear_maps N.to_submodule N.to_submodule :=
+  linear_map.cod_restrict _ (α.to_linear_action.to_endo_map _ _ _) N.lie_mem
+
+instance lie_quotient_action : linear_action R L N.quotient :=
+  linear_action.of_endo_map _ _ _ (linear_map.comp (submodule.mapqₗ N N) lie_submodule_invariant)
+
+lemma lie_quotient_action_apply (z : L) (m : M) :
```

Comment:
You are using two different kind of `mk`. Is that intentional?

Reply:
Yuck, not at all intentional. Well spotted.